content/docs/reference/config/installation-options-version-comparison/index.md

```diff
@@ -0,0 +1,445 @@
+---
+title: Installation Options - Generate differences between release-1.0 and release-1.1
+description: Describes the differences between the current and previous releases of installation options from the Helm charts.
+weight: 30
+keywords: [kubernetes,helm]
+---
+
+The table below shows the comparison of the installation options (used to customize Istio install using Helm) between release 1.0 and release 1.1. These include new installation options that have been added in the new release, installation options that were present in the previous release but removed from the new release and installation options whose values or descriptions have been modified in the current release.
+
+<!-- Run python scripts/tablegen.py to generate this table -->
+
+<!-- AUTO-GENERATED-START -->
+
```

Comment:
can you add a section `## Unmodified configuration options`

Reply:
Begin with the known. The existing options that were not changed.